### PR TITLE
Region dashboard | With mock data

### PR DESCRIPTION
**Pull request by cuteolaf**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,5 +1,101 @@
...
+            variant='subtitle1'
+            sx={{ color: theme.palette.text.primary }}
+          >
+            Regions Dashboard
+          </Typography>
+        </Box>
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
```
Comment (Szegoo) [inline src/pages/index.tsx:41]: Actually, I think we should be able to easily fit two regions in a single row.

File: src/pages/index.tsx [hunk 2]
```diff
@@ -1,5 +1,101 @@
...
+            gap: '2rem',
+            mt: '1rem',
+          }}
+        >
+          {[1, 2].map((_, index) => (
+            <RegionCard
+              key={index}
+              region={{
```
Comment (Szegoo) [inline src/pages/index.tsx:49]: After getting [this](https://github.com/RegionX-Labs/Coretime-Mock/pull/19) merged we can have actual data here. Do you want to make that change in this PR, or do that separately?

Comment (cuteolaf) [reply src/pages/index.tsx:49]: we can remove this block for merging the PR.
it's just for demonstration purpose only.

File: src/components/RegionCard/index.tsx [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+          className={styles.duration}
+          style={{
+            borderColor: theme.palette.grey[200],
+            color: theme.palette.grey[200],
+          }}
+        >
+          <AccessTimeIcon sx={{ fontSize: '1.25em' }} />
+          {`Duration: ${length}`}
+        </div>
+        <Typography variant='subtitle2'>Region 1 </Typography>
```
Comment (Szegoo) [inline src/components/RegionCard/index.tsx:54]: Should read the values from `region`

File: src/components/RegionCard/index.tsx [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        <Box
+          sx={{
+            display: 'flex',
+            flexDirection: 'column',
+            gap: '0.5rem',
+            color: theme.palette.grey[200],
+          }}
+        >
+          <Typography variant='h2'>Begin: 5 days ago</Typography>
+          <Typography variant='h2'>End: 15 days later</Typography>
```
Comment (Szegoo) [inline src/components/RegionCard/index.tsx:64]: The same applies here and the rest of the data in this file.

**Review by Szegoo — CHANGES_REQUESTED** [review]
Looks good, just a few small fixes required.

File: src/components/RegionCard/index.tsx (hunk 4 above)
Comment (cuteolaf) [reply src/components/RegionCard/index.tsx:64]: same

**Review by Szegoo — APPROVED** [review]

**Pull request merged**